tests/integration_tests/performance/test_boottime.py

```diff
@@ -129,6 +129,7 @@ def _configure_and_run_vm(microvm, network=False, initrd=False):
         "vcpu_count": 1,
         "mem_size_mib": 128,
         "boot_args": DEFAULT_BOOT_ARGS + " init=/usr/local/bin/init",
+        "enable_entropy_device": True,
```

Comment:
This could be done in a separate PR as well but I wanted to know if it relating the bootime with entropy events would add any value?
If yes, then would it be ok to call flush_metrics and collect some entropy metrics like entropy_event_count, entropy_bytes or host_rng_fails? We can also try to assert if there was any failure detected in the entropy metrics like entropy_event_fails or activate_fails.

Reply:
ok so does it mean that if for some reason entropy event increase/decrease during boot then relating it to a change in boot time won't make sense at all?

Reply:
I'm saying that they do not necessarily correlate. On x86, for example there shouldn't be any difference.